data.sql

```diff
@@ -31,4 +31,28 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
 VALUES ('Blossom','1998-10-13',17,true,3);
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
-VALUES ('Ditto','2022-05-14',22,true,4);
\ No newline at end of file
+VALUES ('Ditto','2022-05-14',22,true,4);
+
+---->> Lesson: Query multiple tables starts here <<----
+INSERT INTO owners ( full_name, age)
+VALUES ('Sam Smith ',34),
+       ('Jennifer Orwell',19),
+       ('Bob',45),
+       ('Melody Pond',77),
+       ('Dean Winchester ',14),
+       ('Jodie Whittaker',38);
+
+INSERT INTO species (name) 
+VALUES ('Pokemon'),('Digimon');
+
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon%';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+SELECT * FROM animals;
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon','Boarmon');
+
```

Comment:
- [ ] It is required to write queries (using JOIN) to answer the some questions according to the project requirements. Kindly check the project requirements and implement your queries. 

### Project Requirements.
![prj-reqt](https://github.com/Diegogagan2587/vet-clinic-database/assets/26597432/e65bd7f6-58ac-4bf9-8bdf-49ce4c21e256)

Reply:
@Qoosim Hi, Hope you are doing Great! I have wrote the queryes within the `queries.sql` file, those are avaialbe starting on the line 73, Thanks :D!
![image](https://github.com/Diegogagan2587/vet-clinic-database/assets/61764778/66a31cc3-7515-4c22-be63-c395cc48440a)

![image](https://github.com/Diegogagan2587/vet-clinic-database/assets/61764778/fae64ffc-a26c-41ac-b8ac-ce26aa156fcc)

Reply:
@Qoosim Thanks for pointing out the changes I need to do within the Readme, I'll do it now!